builtin.aqua

```diff
@@ -97,6 +101,14 @@ service Kademlia("kad"):
     -- Instructs node to return the locally-known nodes
     -- in the Kademlia neighborhood for a given key
     neighborhood(key: PeerId, already_hashed: bool) -> []PeerId
+    -- Merges given lists and sorts them by distance to target
+    -- Arguments:
+    -- target – base58 string; result is sorted by XOR distance to target
+    -- left – list of base58 strings
+    -- right – list of base58 strings
+    -- count – how many items to return
+    -- Returns: list of base58 strings sorted by distance to target; list will contain at most count elements
+    merge(target: string, left: []string, right: []string, count: u32) -> []string
```

Comment:
can I use `0` count or something like this for `all`?

Reply:
no 🤔

Reply:
If you omit this argument (in AIR), then the default would be 20. Maybe add this to doc?

Reply:
for sure